### PR TITLE
V250

### LINKED ISSUE #120: Validation (2017): V250 (Loan Amount)
**Edit #:** V250
**Type:** Validity
**Category:** Loan/Application Register (only)
**Column:** Loan Amount

Loan amount must be numeric and > zero.

Additional resources:
* [FFIEC's 2016 HMDA File Spec](https://www.ffiec.gov/hmda/pdf/spec2016.pdf)
* [FFIEC's 2016 HMDA Edits](https://www.ffiec.gov/hmda/pdf/edit2016.pdf)
* [CFPB's 2017 HMDA File Spec](http://www.consumerfinance.gov/hmda/static/for-filers/2017/2017-HMDA-File-Specifications.pdf)

### PR DESCRIPTION
**Pull request by kgudel**
closes #120

### CODE REVIEW

File: validation/src/test/scala/hmda/validation/rules/lar/validity/V250Spec.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ LoanApplicationRegister, Loan }
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V250Spec extends LarEditCheckSpec with BadValueUtils {
+  property("Income must be numeric and positive") {
```
Comment (nickgrippin) [inline validation/src/test/scala/hmda/validation/rules/lar/validity/V250Spec.scala]: Might be a bit nit-picky (sorry), but these tests are for loan amounts, not incomes.

File: validation/src/main/scala/hmda/validation/rules/lar/validity/V250.scala [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ CommonDsl, Result }
+import hmda.validation.rules.EditCheck
+
+object V250 extends EditCheck[LoanApplicationRegister] {
+
+  override def apply(lar: LoanApplicationRegister): Result = {
+    (lar.loan.amount is numeric) and (lar.loan.amount is greaterThan(0))
```
Comment (nickgrippin) [inline validation/src/main/scala/hmda/validation/rules/lar/validity/V250.scala]: This is interesting.... The validation specifies that we need to check if `lar.loan.amount` is a number, but this field stores an Int (which is a Number).  Is the check here necessary?

**Comment by schbetsy** [comment]
LGTM; just needs conflicts fixed.

**Comment by nickgrippin** [comment]
LGTM as well

**Pull request merged**